### PR TITLE
Merge Hive 3 branch to main

### PR DESCRIPTION
**Pull request by justineyster**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -2,34 +2,62 @@ FROM azul/zulu-openjdk-debian:11.0.11
...
+# Configure Hadoop AWS Jars to be available to hive
+RUN ln -s ${HADOOP_HOME}/share/hadoop/tools/lib/*aws* ${HIVE_HOME}/lib
 
-RUN zip -q -d /opt/apache-hive-*-bin/lib/log4j-core-*.jar org/apache/logging/log4j/core/lookup/JndiLookup.class
-
-USER hive
+COPY conf ${HIVE_HOME}/conf
+COPY scripts/entrypoint.sh ${HIVE_HOME}/entrypoint.sh
+
+RUN zip -q -d ${HIVE_HOME}/lib/log4j-core-*.jar org/apache/logging/log4j/core/lookup/JndiLookup.class
```
Comment (apreethi13) [inline Dockerfile:33]: do we still need this?

Comment (justineyster) [reply Dockerfile:33]: No, that was a temporary patch until we could upgrade. If you run it now it errors with "nothing to do".

Comment (justineyster) [reply Dockerfile:33]: I don't think it hurts to leave it in.

Comment (justineyster) [reply Dockerfile:33]: But technically no, it's ultimately not doing anything.

Comment (justineyster) [reply Dockerfile:33]: Everything in `${HIVE_HOME}/lib/log4j-*` ends up getting deleted in the next step before the latest log4j is installed.

Comment (justineyster) [reply Dockerfile:33]: Let me know if you'd like me to remove it.

Comment (apreethi13) [reply Dockerfile:33]: Well my point more here is, the docker file is hard to parse as reviewer/user who wants to contribute later point in time.. Suggestion would be to give additional context of what's happening as a comment above each block..

If this is not need then would prefer having it removed/cleaned up..

Comment (justineyster) [reply Dockerfile]: Addressed

**Review by apreethi13 — APPROVED** [review]
lgtm 👍
